src/EstablishmentRoll/EstablishmentRoll.js

```diff
@@ -4,10 +4,9 @@ import { connect } from 'react-redux'
 import axios from 'axios'
 import { setEstablishmentRollData, setApplicationTitle } from '../redux/actions'
 import EstablishmentRollBlock from './EstablishmentRollBlock'
-import Spinner from '../Spinner'
-import Error from '../Error'
+import Page from '../Page/Page'
 
-export class EstablishmentRollContainer extends Component {
+export class EstablishmentRoll extends Component {
```

Comment:
I thought we had a general convention for having container in a container name - just interested in knowing why you renamed it.